recipes-openxt/xenclient/xenclient-dom0-tweaks/xenstore-init

```diff
@@ -2,3 +2,4 @@
 
 # add xenstore entries here
 
+xenstore-write domid 0
```

Comment:
Yea it sounds like it is supposed to be there for all domains but it was missing for dom0. Ack.